src/main/ruby/core/signal.rb

```diff
@@ -48,9 +48,10 @@ module Signal
     Names['CLD'] = Names['CHLD']
   end
 
-  # We want Numbers[SIGCHLD_VALUE] to refer to 'CHLD' not 'CLD',
-  # as CLD is not recognized by `new sun.misc.Signal("CLD")`
+  # We want Numbers[SIGCHLD_VALUE] to refer to CHLD and ABRT not to CLD and IOT,
+  # as CLD and IOT is not recognized by `new sun.misc.Signal(name)`
   Numbers[Names['CHLD']] = 'CHLD'
+  Numbers[Names['ABRT']] = 'ABRT'
```

Comment:
Another way to fix this would be to order them in
https://github.com/graalvm/truffleruby/blob/49a3f29696c7d3888e7b3d9ca8828407e7398191/tool/generate-native-config.rb#L728-L732
But that's more efforts as we'd need to regenerate the config so this is fine.